packages/cli/src/commands/server/server.js

```diff
@@ -108,5 +108,9 @@ export default {
       description: 'Path to the CLI configuration file',
       parse: (val: string) => path.resolve(val),
     },
+    {
+      name: '--disableWatchMode',
```

Comment:
Not sure about the name here. I think we should name it `interactiveMode` and rename this flag to `--disableInteractiveMode`. WDYT @thymikee ?

Reply:
I'd prefer to have `--interactive` flag and use a built-in feature of commanderjs that let us negate the fag with `no-` prefix, so `--no-interactive` would disable it then.

Reply:
Will do.